Jenkinsfile

```diff
@@ -243,10 +243,15 @@ void postFailureFlexcodeScanAndReport(def vulnerabilities){
     email = email + ',Kavitha.Sivagnanam@marklogic.com'
 
     def body=''
-    vulnerabilities.each { msg -> body=body+msg + '<p>' }
-    body = body + '<p>scan report is attached.'
-
-    sendMail email,"<h3>Pipeline FAILED in stage ${STAGE_NAME} <a href=${BUILD_URL}/console> Check Console Output Here</a></h4>  High vulneribilities: <p> $body",false," pipeline FAILED in ${STAGE_NAME} ", 'inventoriesRprt.report.json'
+    if(vulnerabilities && vulnerabilities.size() != 0) {
+        vulnerabilities.each { msg -> body = body + msg + '<p>' }
+        body = 'There are high vulneribilities: <p> ' + body + '<p>scan report is attached.'
+//      sendMail email,"<h3>Pipeline FAILED in stage ${STAGE_NAME} <a href=${BUILD_URL}/console> Check Console Output Here</a></h4> $body",false," pipeline FAILED in ${STAGE_NAME} ",'inventoriesRprt.report.json'
+        sendMail email, "<h3></h4> $body", false, " high vulnerabilities in stage ${STAGE_NAME} ",'inventoriesRprt.report.json'
```

Comment:
Closing tag for <h3> and opening tag for <h4> missed in the body of the email

Reply:
fixed